src/data/context/apyChartContext.tsx

```diff
@@ -166,9 +162,15 @@ export const ApyChartProvider: FC<{
     apy: true,
   })
   const [timeline, setTimeline] = useState<Timeline>("1W")
-  const launchDate = Object.values(cellarDataMap).find(
+  const cellarData = Object.values(cellarDataMap).find(
     (item) => item.config.cellar.address === address
-  )?.launchDate!
+  )
+  const launchDate = cellarData?.launchDate!
+  const [{ data: sgData }] = useGetStrategyDataQuery({
```

Comment:
we have `useStrateyData` please use that instead this subgraph. so it's not refetching